i18n/i18n.go

```diff
@@ -1,32 +1,97 @@
 package i18n
 
 import (
+	"encoding/json"
 	"fmt"
 	"path"
-	"sort"
 	"strings"
 
-	"github.com/nicksnyder/go-i18n/i18n"
-	"github.com/nicksnyder/go-i18n/i18n/language"
+	"github.com/nicksnyder/go-i18n/v2/i18n"
+	"golang.org/x/text/language"
 
 	"github.com/IBM-Cloud/ibm-cloud-cli-sdk/resources"
 )
 
 const (
 	defaultLocale   = "en_US"
-	resourcesSuffix = ".all.json"
+	resourcesSuffix = ".json"
+	resourcesPrefix = "all."
 )
 
-var T i18n.TranslateFunc
+var (
+	bundle *i18n.Bundle
```

Comment:
**Callout**: Added `bundle` variable that will handle initializing the translate function and setting the default langugae.